mc2/controllers/docker/tests/test_docker_controller_hypothesis.py

```diff
@@ -94,7 +94,12 @@ def check_and_clear_appdata(appdata, controller):
     assert appdata.pop("cpus") == controller.marathon_cpus
     assert appdata.pop("mem") == controller.marathon_mem
     assert appdata.pop("instances") == controller.marathon_instances
-    check_and_remove_optional(appdata, "cmd", controller.marathon_cmd)
+
+    if controller.marathon_args:
+        check_and_remove_optional(appdata, "args", [controller.marathon_args])
```

Comment:
This list seems a bit odd here, should args be multiple fields or do some kind of validation so that the underlying field data is exposed as a list?